src/styles/components/_SignupPage.scss

```diff
@@ -26,16 +26,16 @@ h2 {
 }
 
 .signup-form-input::placeholder { /* Chrome/Opera/Safari */
-  color: $purple;
+  color: #c4c4c4;
```

Comment:
Why did you remove the variable? Put it back please

Reply:
I have used a variable.